api/pytest.ini

```diff
@@ -1,2 +1,8 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = catalog.settings
+
+pythonpath = .
+
+filterwarnings=
+  ignore:Unverified HTTPS request
+  ignore:DateTimeField AccessToken\.expires received a naive datetime
```

Comment:
Do we have an explanation for why these warnings are filtered? Perhaps like the catalog's documentation for those warnings

Reply:
I don't know. Maybe make a new issue to audit and document them?